students/alex_whitty/lesson2/print_grid2.py

```diff
@@ -0,0 +1,51 @@
+"""Wrote a function that takes two arguments and prints a grid"""
+
+
+def plus():
+    plus = '+'
+    return plus
+
+
+def minus(m, args):
+    minus = ' - ' * args
+    return minus
+
+
+def vert_dash(v, args):
+    vert_dash = '|' + ' ' * args + ' | ' + ' ' * args + '|'
+    return vert_dash
+
+
+def print_grid(v, m):
+    print(plus() + minus(1, 1) + plus() + minus(1, 1) + plus())
+    print(vert_dash(2, 2))
+    print(plus() + minus(1, 1) + plus() + minus(1, 1) + plus())
+    print(vert_dash(2, 2))
+    print(plus() + minus(1, 1) + plus() + minus(1, 1) + plus())
+
+
+
+
+print_grid(2,1)
```

Comment:
the variables your passing aren't actually being used in your program.  You should swap out your `2`s and `1`s for the variables.